Client/Frontend/Browser/TopTabsViewController.swift

```diff
@@ -107,9 +107,9 @@ class TopTabsViewController: UIViewController {
     required init?(coder aDecoder: NSCoder) {
         fatalError("init(coder:) has not been implemented")
     }
-    
-    override func viewWillAppear(_ animated: Bool) {
-        super.viewWillAppear(animated)
+
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
         self.reloadData()
```

Comment:
I find it odd that the view isn't created when `viewWillAppear` is called. Moving the reload to `viewDidAppear` seems to late. Would calling `reloadData` from `viewDidLayoutSubviews` work better here?